TestLogin/TestLogin/TestLogin/MainViewModel.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using Xamarin.Forms;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TestLogin
+{
+    internal class MainViewModel : INotifyPropertyChanged
+    {
+       
+        #region
+        public string _username;
+        public string _password;
+        #endregion
+
+
+        #region
+        public string Username
+        {
+            get
+            {
+                return _username;
+            }
+            set
+            {
+                _username = value;
+                OnPropertyChanged("Username");
+                OnPropertyChanged(nameof(Username));
+                OnPropertyChanged(nameof(DisplayName));
+
+            }
+        }
+        public string Password
+        {
+            get
+            {
+                return _password;
+            }
+            set
+            {
+                _password = value;
+                OnPropertyChanged("Password");
+                OnPropertyChanged(nameof(Password));
+                OnPropertyChanged(nameof(Status));
+
+            }
+
+        }
+        public Command Login 
```

Comment:
It's better if u rename this like LoginCommand, So we can easily identify its a command

Reply:
Sure. Noted

Reply:
Changed to LoginCommand